CMakeLists.txt

```diff
@@ -410,6 +410,12 @@ endif()
 add_subdirectory(Tools/JSObjects "${CMAKE_CURRENT_BINARY_DIR}/JSObjects")
 target_link_libraries(ds2 PRIVATE jsobjects)
 
+if(APPLE OR ANDROID OR LINUX)
+  # include all symbols on debug builds to print a readable backtrace on ds2 crash
+  target_link_options(ds2 PRIVATE
+    $<$<OR:$<CONFIG:Debug>,$<CONFIG:RelWithDebInfo>>:-rdynamic>)
+endif()
```

Comment:
What is the size implication for this? Load time implications? I don't think that we should be doing this, but instead be using the split debug information to symbolicate the stack trace.

Reply:
I was also concerned about the size and performance, and that's what lead me to enable it only on Debug+RelWithDebInfo. Symbolication on the build machine is definitely better. I'll figure out how to do that and remove this part of the PR.

Reply:
We distribute the necessary tools for that in the toolchain. `llvm-symbolizer` should do the trick (it is the equivalent to `atos` on macOS.